src/BibleWell.App/App.axaml.cs

```diff
@@ -33,22 +35,33 @@ public override void OnFrameworkInitializationCompleted()
 
             desktop.MainWindow = new MainWindow
             {
-                DataContext = mainViewModel,
+                DataContext = mainViewModel
             };
         }
         else if (ApplicationLifetime is ISingleViewApplicationLifetime singleViewPlatform)
         {
-            singleViewPlatform.MainView = new MainView
+            singleViewPlatform.MainView = new OpenFileView
             {
-                DataContext = mainViewModel,
+                DataContext = mainViewModel
             };
         }
 
+        Ioc.Default.ConfigureServices(serviceProvider);
         base.OnFrameworkInitializationCompleted();
     }
 
-    internal static void ConfigureServices(IServiceCollection services)
+    internal static void ConfigureServices(IServiceCollection services, IApplicationLifetime? applicationLifetime)
     {
+        if (applicationLifetime is IClassicDesktopStyleApplicationLifetime desktop)
+        {
+            services.AddSingleton<IFilesService>(x => new FilesService(desktop.MainWindow));
+        }
+        else if (applicationLifetime is ISingleViewApplicationLifetime singleViewPlatform)
+        {
+            services.AddSingleton<IFilesService>(x => new FilesService(singleViewPlatform.MainView));
+        }
+
```

Comment:
@kaseywright this is the part that feels a little wonky to me. It'd be nice for either the Service to take some sort of interface instead of the concrete Visual, so maybe trying out the maui essentials will mitigate it. 

Of course, I could be seeing this with too critical of an eye. Any thoughts?